webapp/src/Components/Replay/ReplayViewer/ThreeScene.jsx

```diff
@@ -38,27 +54,128 @@ export class ThreeScene extends Component {
             this.frameId = requestAnimationFrame(this.animate)
         }
     };
+
     stop = () => {
         cancelAnimationFrame(this.frameId)
     };
+
     animate = () => {
-        this.cube.rotation.x += 0.01;
-        this.cube.rotation.y += 0.01;
+        this.updateBall();
+        this.updatePlayers();
+        this.updateCamera();
```

Comment:
I would also like this separate from the generation of the models as this will be specific to each mode.

Shots will animate differently than the normal replays (most likely)